docs/ldap-setup.md

```diff
@@ -4,20 +4,11 @@ This document describes an example configuration of LDAP authentication and auth
 
 ## Assumptions
 
-1. The setup of an LDAP server is out of scope of this document. We assume that you are familiar with the LDAP server schema.
-
-2. You have the LDAP server up and running and it is accessible to the servers with Percona Server for MongoDB installed.
-
-3. This document primarily focuses on OpenLDAP used as the LDAP server and the examples are given based on the OpenLDAP format. If you are using Active Directory, refer to the [Active Directory configuration](#active-directory-configuration) section.
-
-4. You have the `sudo` privilege to the server with the Percona Server for MongoDB installed.
-
-## Prerequisites
-
-* In this setup we use anonymous binds to the LDAP server. If your LDAP server disallows anonymous binds, create the user that Percona Server for MongoDB will use to connect to and query the LDAP server. Define this user’s credentials for the `security.ldap.bind.queryUser` and `security.ldap.bind.queryPassword`  parameters in the `mongod.conf` configuration file.
-
-
-* In this setup, we use the following OpenLDAP groups:
+* The setup of an LDAP server is out of scope of this document. We assume that you are familiar with the LDAP server schema.
+* An explanation of [RFC4514](https://www.ietf.org/rfc/rfc4514.txt), [RFC4515](https://tools.ietf.org/html/rfc4515), [RFC4516](https://tools.ietf.org/html/rfc4516), or LDAP queries is out of scope of this setup. Please review the RFC directly or use your preferred LDAP resource.
```

Comment:
I would trade places by mentioning LDAP queries first and then RCFs that describe special character escaping in those queries